app/controllers/users_controller.rb

```diff
@@ -53,7 +53,7 @@ def choose_identity
   
   def update_identity
     new_identity = h(params[:identity].to_s)
-    return redirect_to choose_identity_path unless %w(Student Volunteer Mentor).include?(new_identity)
+    return redirect_to choose_identity_path unless %w(Participant Volunteer Mentor).include?(new_identity)
```

Comment:
Hey Matt, We are planning to drop the usage of the type "Student". What's your idea on this? @mattharris5